diff --git a/out/zqlite/src/table-source.js b/out/zqlite/src/table-source.js
index c314323c0d861809314faf96b2040c4e99cf55e7..6f73d6c823b81139333e76c08ac4f94235dfedab 100644
--- a/out/zqlite/src/table-source.js
+++ b/out/zqlite/src/table-source.js
@@ -488,7 +488,7 @@ function fromSQLiteType(valueType, v, column) {
         case 'null':
             if (typeof v === 'bigint') {
                 if (v > Number.MAX_SAFE_INTEGER || v < Number.MIN_SAFE_INTEGER) {
-                    throw new UnsupportedValueError(`value ${v} (in column ${column}) is outside of supported bounds`);
+                    // throw new UnsupportedValueError(`value ${v} (in column ${column}) is outside of supported bounds`);
                 }
                 return Number(v);
             }
