diff --git a/out/zqlite/src/table-source.js b/out/zqlite/src/table-source.js
index 19275509339001207344aef650f8e40f2dc1902f..c22bde3566e5c636ef9e36d42213aeb2ccbec5f7 100644
--- a/out/zqlite/src/table-source.js
+++ b/out/zqlite/src/table-source.js
@@ -490,7 +490,8 @@ function fromSQLiteType(valueType, v, column) {
         case 'null':
             if (typeof v === 'bigint') {
                 if (v > Number.MAX_SAFE_INTEGER || v < Number.MIN_SAFE_INTEGER) {
-                    throw new UnsupportedValueError(`value ${v} (in column ${column}) is outside of supported bounds`);
+                    // throw new UnsupportedValueError(`value ${v} (in column ${column}) is outside of supported bounds`);
+                  return Number(v);
                 }
                 return Number(v);
             }
